src/kakarot/interfaces/interfaces.cairo

```diff
@@ -90,6 +90,11 @@ namespace IAccount {
 
     func set_authorized_pre_eip155_tx(msg_hash: Uint256) {
     }
+
+    func execute_starknet_call(
+        called_address: felt, function_selector: felt, calldata_len: felt, calldata: felt*
```

Comment:
```suggestion
        contract_address: felt, function_selector: felt, calldata_len: felt, calldata: felt*
```

to use the same signature as the syscall `call_contract`

Reply:
`contract_address` is a reserved keyword, as it is the the first, required argument when calling of `IContract.call(contract_address)`

Thus, we can't use `contract_address`

Reply:
maybe `to` like in the `Call` struct

```
struct Call {
    to: felt,
    selector: felt,
    calldata_len: felt,
    calldata: felt*,
}
```

I just don't really like the passive `called_` ton